### DISCUSSION TITLE
"Unknown error" when reseting a forgotten password

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16758] [Archive]
In fact, 99% of the time it's because the user violates the password policy.
It is understandable that, for security reasons, this policy is not publicly available.

But the lack of "Unknown Error" information leads the user to think that it is the system that is failing, not him.

Would it be possible to customize the error message of `/admin/reset-password`?

Related to
 #12788
 #8946
a year old.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/12788
